src/core_modules/capture-core/components/WorkingLists/EventWorkingLists/epics/getEventListData.js

```diff
@@ -177,6 +177,11 @@ export const getEventListData = async (
     categoryCombinationId?: ?string,
 ) => {
     const mainColumns = getMainColumns(columnsMetaForDataFetching);
+    // Remove programStageId from the API if program is single event
+    if (queryArgs?.programStageId === 'EventProgramStage') {
+        queryArgs.programStageId = undefined;
```

Comment:
Hey, 
I am curious to know why you don't remove `programStageId` from the object and prefer to use `undefined`?  
Thanks!

Reply:
Just a matter of old habits I think. Changed it now, thanks! 👍